Configuration/Eras/python/Era_Run2_HI_cff.py

```diff
@@ -2,7 +2,7 @@
 
 from Configuration.Eras.Modifier_run2_common_cff import run2_common
 from Configuration.Eras.Modifier_run2_HI_specific_cff import run2_HI_specific
-from Configuration.Eras.Modifier_stage1L1Trigger_cff import stage1L1Trigger
+from Configuration.Eras.Modifier_stage2L1Trigger_2017_cff import stage2L1Trigger_2017
 
-Run2_HI = cms.ModifierChain(run2_common, run2_HI_specific, stage1L1Trigger)
+Run2_HI = cms.ModifierChain(run2_common, run2_HI_specific, stage2L1Trigger_2017)
```

Comment:
this era is still supposed to work with 2015 HI data.
Is the "2017" here simply nominal and will still work on that data?

Reply:
@slava77 
In that case we better use stage2L1Trigger modifier.

Reply:
Will provide the change and retest.

Reply:
but did we have the stage 2 trigger in 2015? @rekovic